test/integration/cli-image-create-kvm.test.js

```diff
@@ -157,13 +181,23 @@ test('triton image ...', testOpts, function (suite) {
     // have a way to know if the attempt failed or if it is just taking a
     // really long time.
     suite.test('  cleanup: triton rm', {timeout: 10 * 60 * 1000}, function (t) {
+        if (!originInst) {
+            t.comment('Cannot create original instance. Skip');
+            t.end();
+            return;
+        }
         h.safeTriton(t, ['rm', '-w', originInst.id, derivedInst.id],
                 function () {
             t.end();
         });
     });
 
     suite.test('  cleanup: triton image rm', function (t) {
+        if (!originInst) {
+            t.comment('Cannot create image. Skip');
```

Comment:
same

Reply:
And done! I think that's everyone of them.